haas/model.py

```diff
@@ -122,115 +122,12 @@ class Node(Model):
-            '-P', self.ipmi_pass,
-            '-H', self.ipmi_host] + args)
-        if status != 0:
-            logger = logging.getLogger(__name__)
-            logger.info('Nonzero exit status from ipmitool, args = %r', args)
-        return status
-
-    @no_dry_run
-    def power_cycle(self):
-        """Reboot the node via ipmi.
-
-        Returns True if successful, False otherwise.
-        """
-        self._ipmitool(['chassis', 'bootdev', 'pxe'])
-        if self._ipmitool(['chassis', 'power', 'cycle']) == 0:
-            return
-        if self._ipmitool(['chassis', 'power', 'on']) == 0:
-            # power cycle will fail if the machine isn't running, so let's
-            # just turn it on in that case. This way we can save power by
-            # turning things off without breaking the HaaS.
-            return
-        # If it still doesn't work, then it's a real error:
-        raise OBMError('Could not power cycle node %s' % self.label)
-
-    @no_dry_run
-    def power_off(self):
-        if self._ipmitool(['chassis', 'power', 'off']) != 0:
-            raise OBMError('Could not power off node %s', self.label)
-
-    @no_dry_run
-    def start_console(self):
-        """Starts logging the IPMI console."""
-        # stdin and stderr are redirected to a PIPE that is never read in order
-        # to prevent stdout from becoming garbled.  This happens because
-        # ipmitool sets shell settings to behave like a tty when communicateing
-        # over Serial over Lan
-        Popen(
-            ['ipmitool',
-            '-H', self.ipmi_host,
-            '-U', self.ipmi_user,
-            '-P', self.ipmi_pass,
-            '-I', 'lanplus',
-            'sol', 'activate'],
-            stdin=PIPE,
-            stdout=open(self.get_console_log_filename(), 'a'),
-            stderr=PIPE)
-
-    # stdin, stdout, and stderr are redirected to a pipe that is never read
-    # because we are not interested in the ouput of this command.
-    @no_dry_run
-    def stop_console(self):
-        call(['pkill', '-f', 'ipmitool -H %s' %self.ipmi_host])
-        proc = Popen(
-            ['ipmitool',
-            '-H', self.ipmi_host,
-            '-U', self.ipmi_user,
-            '-P', self.ipmi_pass,
-            '-I', 'lanplus',
-            'sol', 'deactivate'],
-            stdin=PIPE,
-            stdout=PIPE,
-            stderr=PIPE)
-        proc.wait()
-
-    def delete_console(self):
-        if os.path.isfile(self.get_console_log_filename()):
-            os.remove(self.get_console_log_filename())
-
-    def get_console(self):
-        if not os.path.isfile(self.get_console_log_filename()):
-            return None
-        with open(self.get_console_log_filename(), 'r') as log:
-            return "".join(i for i in log.read() if ord(i)<128)
-
-    def get_console_log_filename(self):
-        return '/var/run/haas_console_logs/%s.log' % self.ipmi_host
+    # The node is initially registered with no nics; see the Nic class.
```

Comment:
This seems like an odd place to put this -- a random comment at the end of a class definition? Two points:

1. I'm not sure what this has to do with this change.
2. If you think this needs clarification, there's probably a more sensible place to put it, maybe the api docs for `node_register`?

Despite (1) I don't really mind you slipping more docs in if you want, but put them in a more reasonable place.